mailbox/plugin/quota-search/src/test/java/org/apache/james/quota/search/QuotaClauseTest.java

```diff
@@ -118,4 +118,13 @@ public void andClauseShouldInstanciate() {
         assertThat(andQuery.getClauses())
             .containsExactly(first, second);
     }
+
+    @Test
+    public void nestedAndClausesAreNotSupported() {
```

Comment:
are then :)

Reply:
Sorry, I don't get that one. Can you be more explicit?

Reply:
nothing special, just that as you are using "are" here you can s/is/are in the previous comment

Reply:
Ok, so I don't need to do anything here and care about this comment, right?